_docs/_latest/develop/dev/testing.md

```diff
@@ -59,3 +59,18 @@ If you want to execute a subset of tests (e.g., all tests in a specific subdirec
     ```bash
     docker -l error exec -it build /bin/bash -c "python -m unittest discover -s tests/common -p '*_test.py'"
     ```
+
+### Continual Testing
```

Comment:
This section heading can be more clear, with something along the line `Update Code and Re-Run Test`.